repo.json

```diff
@@ -198,7 +247,7 @@
             "display_name": "WarehouseAssets",
             "license": "Apache-2.0 or MIT",
             "license_url": "https://opensource.org/licenses/Apache-2.0",
-            "origin": "The name of the originator or creator",
+            "origin": "RobotecAI",
```

Comment:
Changing this here, instead of in `versions_data` will likely break older versions of the Gem because the O3DE CLI will run a sha check and compare all the gem fields and error if it finds any inconsistencies between the repo.json and what appears in gem.json